### PR TITLE
Phone scss tidy

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

* a general clean-up of the _phone.scss file
* minor design changes where bugs were spotted or when @yaili spotted something
 * bug on helping hand section on /phone/developer image
  * bug on form on /phone/devices medium 
image small 
image
* removed unused _phone_promo.html file
* used columns for better medium and small screen display of form in _devices_newsletter.html 

## QA

1. go to every page in the cloud section and see that it looks nearly identical to live

### CODE REVIEW

**Comment by anthonydillon** [comment]
## QA
Large screen - /phone/developers
There is vertical scrolling:
for developers ubuntu for phones ubuntu

File: static/css/section/_phone.scss [hunk 1]
```diff
@@ -1,924 +1,612 @@
...
-}
+    .row-hero {
+      background-size: cover;
+      color: $cool-grey;
+      margin-top: 0;
+      overflow: hidden;
+      -moz-transition-duration: .7s;
+      -o-transition-duration: .7s;
+      -webkit-transition-duration: .7s;
+      transition-duration: .7s;
```
Comment (anthonydillon) [inline static/css/section/_phone.scss]: No need for `transition-duration` if there is no transitions

File: static/css/section/_phone.scss [hunk 2]
```diff
@@ -1,924 +1,612 @@
...
+          margin-top: 80px;
+        }
+      }
 
-    .the-video {
-      .close-video {
-        display: none;
+      .intro {
+        @media only screen and (min-width : $breakpoint-medium) {
+          font-size: 23px;
```
Comment (anthonydillon) [inline static/css/section/_phone.scss]: Would be nice to start `rem`'ing these values.

File: static/css/section/_phone.scss [hunk 3]
```diff
@@ -1,924 +1,612 @@
...
 
-        h1 {
-          padding-top: 0;
-        }
-    }
-    #main-content .row-hero .for-mobile {
-        margin: 0 -10px 10px;
-    }
+      @media only screen and (min-width: $breakpoint-medium) {
+        // ANT - WHY IS THIS OVERWRITTEN BY .tablet .row, .phone .row { padding: 40px 10px 20px; } ON LIVE?
```
Comment (anthonydillon) [inline static/css/section/_phone.scss]: No idea. Might not be needed anymore.

File: static/css/section/_phone.scss (hunk 1 above)
Comment (pmahnke) [inline static/css/section/_phone.scss]: removed

File: static/css/section/_phone.scss (hunk 3 above)
Comment (pmahnke) [inline static/css/section/_phone.scss]: agreed... looks better without... removing

File: static/css/section/_phone.scss (hunk 2 above)
Comment (pmahnke) [inline static/css/section/_phone.scss]: fixed.

**Pull request merged**